README.md

```diff
@@ -7,6 +7,62 @@ This package is designed to work with **Python 2.6+** and **Python 3.3+**
 ***This package is an early work in progress under active development, and is not (yet)
 considered a complete implementation of Ion.***
 
+## Development
+It is recommended to use `virtualenv` to create a clean environment to build/test Ion Python.
+
+```
+$ virtualenv venv
+...
+$ . venv/bin/activate
+$ pip install -r requirements.txt
+$ pip install -e .
+```
+
+You can also run the tests through `setup.py` or `py.test` directly.
+
+```
+$ python setup.py test
+```
+
+### Tox Setup
+In order to verify that all platforms we support work with Ion Python, we use a combination
+of [tox](http://tox.readthedocs.io/en/latest/) with [pyenv](https://github.com/yyuu/pyenv).
+
+Install relevant versions of Python and set them up for local usage:
+
+```
+$ for V in 2.6.9 2.7.12 3.3.6 3.4.5 3.5.2 pypy-5.3.1; do pyenv install $V; done
+...
+$ pyenv local 2.6.9 2.7.12 3.3.6 3.4.5 3.5.2 pypy-5.3.1
+```
+
+Note that on Mac OS X, you may need to change the `CFLAGS`:
+
+```
+$ for V in 2.6.9 2.7.12 3.3.6 3.4.5 3.5.2 pypy-5.3.1; do
+    CFLAGS="-I$(xcrun --show-sdk-path)/usr/include" pyenv install $V; done
+```
+
+Once you have these installations, add them as a local `pyenv` configuration
+
+```
+$ pyenv local 2.6.9 2.7.12 3.3.6 3.4.5 3.5.2 pypy-5.3.1
+```
+
+Assuming you have `pyenv` properly set up (making sure `pyenv init` is evaluated into your shell),
+you can now run `tox`:
+
+```
+# Run tox for all versions of python which executes py.test.
+$ tox
```

Comment:
This sentence could benefit from a comma or two. I think it means this:

"Run tox, which executes py.test, for all versions of python."